custom/ewsghana/reports/__init__.py

```diff
@@ -169,3 +172,31 @@ def is_reporting_type(self):
             if not location_type.administrative
         ]
         return sql_location.location_type in reporting_types
+
+    @property
+    def export_table(self):
+        r = self.report_context['reports'][0]['report_table']
+        return [self._export_table(r['title'], r['headers'], r['rows'])]
+
+    # Export for Facility Page Report, which occurs in every multireport
+    def _export_table(self, export_sheet_name, headers, formatted_rows, total_row=None):
+        def _unformat_row(row):
+            return [col.get("sort_key", col) if isinstance(col, dict) else col for col in row]
+
+        table = headers.as_export_table
+        rows = [_unformat_row(row) for row in formatted_rows]
+        # Removing html icon tag from MOS column
```

Comment:
i was wondering if this is what you were doing. you can use this function that @kaapstorm recently added: https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/reports/generic.py#L836